### PR TITLE
Fix IPTV Manager timeout issues

### PR DESCRIPTION
**Pull request by dagwieers**
This PR includes:
- A fix to open the socket to IPTV Manager as quickly as possible
- Fix the timezone offset information so IPTV Simple add-on loads the EPG fully

This fixes add-ons/service.iptv.manager#23

### CODE REVIEW

**Comment by dagwieers** [comment]
@michaelarnauts We are still getting a timeout:
```log
2020-05-25 02:20:47.589 T:1937357248 WARNING: CGUIMediaWindow::OnMessage - updating in progress
2020-05-25 02:21:05.635 T:899670912   ERROR: [service.iptv.manager] [resources.lib.modules.addon] Something went wrong while calling plugin.video.yelo: Timout waiting for reply on port 34737
```

**Comment by dagwieers** [comment]
It appears that whatever timeout value is set, the actual timeout for receiving data is 10 seconds.
In the log below it was set to 60 seconds.

```
2020-05-26 04:10:24.861 T:1586578304 WARNING: [service.iptv.manager] [resources.lib.modules.addon] Bound on port 36363...
2020-05-26 04:10:24.862 T:1586578304 WARNING: [service.iptv.manager] [resources.lib.modules.addon] Waiting for a connection from plugin.video.yelo on port 36363...
2020-05-26 04:10:25.743 T:1531102080   ERROR: No handlers could be found for logger "iptv-manager"
2020-05-26 04:10:25.745 T:1586578304 WARNING: [service.iptv.manager] [resources.lib.modules.addon] Connected to 127.0.0.1:37492! Waiting for result...
2020-05-26 04:10:35.796 T:1586578304 WARNING: [service.iptv.manager] [resources.lib.modules.addon] Closing socket on port 36363
2020-05-26 04:10:35.797 T:1586578304   ERROR: [service.iptv.manager] [resources.lib.modules.addon] Something went wrong while calling plugin.video.yelo: Timout waiting for reply on port 36363
```

**Comment by dagwieers** [comment]
@shycoderX This PR still includes fixes we need.

File: resources/lib/iptvmanager.py [hunk 1]
```diff
@@ -10,26 +10,35 @@
...
+
+        def send(self):
+            """Decorator to send over a socket"""
+            import json
+            import socket
+            _LOGGER.debug('Sending data output to IPTV Manager using port %s', self.port)
+            sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+            sock.connect(('127.0.0.1', self.port))
+            try:
+                sock.send(json.dumps(func(self)))  # pylint: disable=not-callable
```
Comment (michaelarnauts) [inline resources/lib/iptvmanager.py:31]: For python 3 support, you need to .encode() the output of json.dumps()

**Comment by dagwieers** [comment]
@michaelarnauts I wanted to resolve the conflicts in this branch, but @shycoderX apparently did already add commits with these changes. So this PR is no longer needed.

**Pull request closed without merging**